ajax/libs/wdt-loading/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "wdt-loading",
+  "filename": "wdtLoading.min.js",
+  "version": "0.1.0",
+  "title": "wdtLoading - Application loading screen",
+  "description": "Asana like application loading screen with custom phrases support",
+  "homepage": "http://ned.im/wdtLoading",
+  "keywords": [
+    "loading",
+    "spinner",
+    "plugin",
+    "application",
+    "javascript",
+    "wdt"
+  ],
+  "author": {
+    "name": "Nedim Arabacı",
+    "url": "http://github.com/needim/"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/needim/wdtLoading.git"
+  },
+  "npmName": "wdt-loading",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "wdtLoading.*+(js|css)"
```

Comment:
Wrong pattern here.
`"wdtLoading.*+(js|css)"`

BTW, I'll like to also add "map" with "js" or "css" so that once the upstream have map file for css or js files, we don't need to manually update the config.